### PR TITLE
LIVE-5008 Remove unnecessary zcash package

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

Remove unnecessary zcash package

### ❓ Context

- **Impacted projects**: `` LLC
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/browse/LIVE-5008

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/package.json [hunk 1]
```diff
@@ -278,6 +277,7 @@
     "jest-environment-jsdom": "28",
     "jest-file-snapshot": "^0.5.0",
     "long": "^4.0.0",
+    "mkdirp": "*",
```
Comment (haammar-ledger) [inline libs/ledger-live-common/package.json:280]: I don't know what this package is for, but it doesn't seemed used anywhere?

**Review by haammar-ledger — COMMENTED** [review]
Can you please also update the `libs/ledger-live-common/dependencies.md` file?

File: libs/ledger-live-common/package.json (hunk 1 above)
Comment (hedi-edelbloute) [reply libs/ledger-live-common/package.json:280]: Looks like it is used in the bot `libs\ledger-live-common\src\bot\portfolio\process-main.ts`

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledger-live-common/package.json (hunk 1 above)
Comment (haammar-ledger) [reply libs/ledger-live-common/package.json:280]: :open_mouth: didn't see it when I searched last time, weird

**Pull request merged**